kitsune/sumo/utils.py

```diff
@@ -350,3 +351,37 @@ def check_for_spam_content(data):
     has_links = has_blocked_link(data)
 
     return is_toll_free or is_nanp_number or has_links
+
+
+WEBPACK_SHORT_PATH_MAPPING = {
+    "protocol/": "node_modules/@mozilla-protocol/core/protocol/",
+    **{
+        f"{app}/": f"kitsune/{app}/static/{app}/"
+        for app in [
+            app.removeprefix("kitsune.")
+            for app in settings.INSTALLED_APPS
+            if app.startswith("kitsune.")
+        ]
+    },
+}
+_webpack_static_cache = {}
```

Comment:
Let's remove caching for now from this PR. Let's have an issue to further investigate this.

Reply:
Along with this let's remove the setting too since it won't be used.

Reply:
https://github.com/mozilla/sumo-project/issues/874